tests/test_create.py

```diff
@@ -1127,38 +1127,6 @@ def test_allow_softlinks(
         assert (prefix / "fonts" / "Inconsolata-Bold.ttf").is_symlink()
 
 
-@pytest.mark.skipif(on_win, reason="nomkl not present on windows")
-def test_remove_features(
```

Comment:
This is now in `test_features` under a new name, right?

Reply:
Correct, moved it and expanded on it